src/main/java/net/id/incubus_core/recipe/IncubusRecipe.java

```diff
@@ -17,14 +14,5 @@ public interface IncubusRecipe<C extends Inventory> extends Recipe<C> {
     IncubusRecipeType<?> getType();
 
     List<IngredientStack> getInputs();
-
-    // TODO - Should this method ignore the DynamicRegistryManager?
-    @Override
-    default String getGroup() {
-        return Registries.ITEM.getId(getOutput(null).getItem()).getPath();
-    }
-
-    default List<ItemStack> getOutputs(DynamicRegistryManager dynamicRegistryManager) {
-        return Collections.singletonList(getOutput(dynamicRegistryManager));
-    }
+    
```

Comment:
Curious to why these were removed - is IncubusRecipe still used anywhere?

Reply:
Recipe does not have getGroup() and getOutputs() anymore, it seems like. Those two are better to be overridden by implementations either way.